### PR TITLE
enh(release): Add Centreon Web 21.10.11 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [x] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 22.10.x (next)

### CODE REVIEW

File: versioned_docs/version-21.10/releases/centreon-core.md [hunk 1]
```diff
@@ -17,6 +17,41 @@ If you have feature requests or want to report a bug, please go to our
...
+#### Bug fixes
+
+- [Configuration] Fixed export of RRDcached path in Centreon Broker configuration
+
+#### Security fixes
+
+- [Administration] Sanitized and bound media import queries
+- [CLAPI] Sanitized and bound Centreon hostgroup class queries
+- [CLAPI] Sanitized and bound Centreon service class queries
+- [CLAPI] Sanitized and bound LDAP listing queries
```
Comment (thiuyendang) [inline versioned_docs/version-21.10/releases/centreon-core.md:31]: [Configuration] Fixed export of RRDcached path in Centreon Broker configuration

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md [hunk 2]
```diff
@@ -16,6 +16,41 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+#### Bug fixes
+
+- [Configuration] Fixed export of RRDcached path in Centreon Broker configuration
+
+#### Security fixes
+
+- [Administration] Sanitized and bound media import queries
+- [CLAPI] Sanitized and bound Centreon hostgroup class queries
+- [CLAPI] Sanitized and bound Centreon service class queries
+- [CLAPI] Sanitized and bound LDAP listing queries
```
Comment (thiuyendang) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md:30]: [Configuration] Fixed export of RRDcached path in Centreon Broker configuration

**Review by thiuyendang — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md [hunk 3]
```diff
@@ -16,6 +16,41 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
 
 ## Centreon Web
 
+### 21.10.11
+
```
Comment (tuntoja) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md:20]:
```suggestion

Release date: `September 30, 2022`

```

File: versioned_docs/version-21.10/releases/centreon-core.md [hunk 4]
```diff
@@ -17,6 +17,41 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.10.11
+
```
Comment (tuntoja) [inline versioned_docs/version-21.10/releases/centreon-core.md:21]:
```suggestion

Release date: `September 30, 2022`

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by thiuyendang — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by thiuyendang — APPROVED** [review]

**Pull request merged**